github/Workflow.pyi

```diff
@@ -2,21 +2,26 @@ from datetime import datetime
 from typing import Any, Dict, Union
 
 from github.Branch import Branch
+from github.Commit import Commit
 from github.GithubObject import CompletableGithubObject, _NotSetType
 from github.NamedUser import NamedUser
 from github.PaginatedList import PaginatedList
+from github.Tag import Tag
 from github.WorkflowRun import WorkflowRun
 
 class Workflow(CompletableGithubObject):
     def __repr__(self) -> str: ...
     def _initAttributes(self) -> None: ...
     def _useAttributes(self, attributes: Dict[str, Any]) -> None: ...
+    def create_dispatch(
+        self, ref: Union[str, Branch, Commit, Tag], inputs: Union[dict, None]
```

Comment:
Can you expand dict to Dict[] with some types for the keys and values?

Reply:
Based on my reading of the Github docs, I am assuming that the values can be one of str, int or Float. 
I have updated it to read `inputs: Union[Dict[str, Union[str, int, float]], _NotSetType]`